x-pack/plugins/fleet/server/services/agent_policy.ts

```diff
@@ -694,10 +694,14 @@ class AgentPolicyService {
           }
         );
       try {
-        await packagePolicyService.runPostDeleteExternalCallbacks(deletedPackagePolicies);
+        await packagePolicyService.runPostDeleteExternalCallbacks(
```

Comment:
Should this one be moved to the package policy service too?

Reply:
Ah, yes!

Reply:
Btw. it is pretty odd to me that we have broken out the delete callbacks to separate handlers (`runPostDeleteExternalCallbacks`, `runDeleteExternalCallbacks`) and call them directly instead of calling `runExternalCallbacks`.